### DISCUSSION TITLE
How to set WIDTH and HEIGHT of map?

### DISCUSSION THREAD

**Discussion by loremru** [maplibre/maplibre-gl-js#3571] [Q&A] [answered]
Maplibre has a great functionality, but I can't find out how to set up just width and height of map...
pixelRatio doesnt work
Can anyone help?...

css doesn't work too

<img width="1219" alt="image" src="https://github.com/maplibre/maplibre-gl-js/assets/66596500/58375ea1-b5fa-423f-95a8-fb1078e8f48e">

**Comment by HarelM** [comment]
Use CSS for the relevant div.

**Comment by loremru** [comment, marked as answer]
I should init map when container has height and width
